res/css/views/rooms/_EventBubbleTile.pcss

```diff
@@ -518,6 +518,19 @@ limitations under the License.
         }
     }
 
+    &.mx_EventTile_info {
+        .mx_EventTile_line {
+            /* timestamp wrapper anchor */
+            > a {
+                inset-inline-start: -70px; /* Add padding to the left side of timestamp wrapper anchor */
+            }
```

Comment:
I don't really understand what this is doing, either. it's somehow overriding the `left: -77px` below?

Reply:
Basically `left: -77px` is included in the selectors below:

````
.mx_EventTile.mx_EventTile_bubbleContainer[data-layout="bubble"],
.mx_EventTile.mx_EventTile_leftAlignedBubble[data-layout="bubble"],
.mx_EventTile.mx_EventTile_info[data-layout="bubble"],
.mx_GenericEventListSummary[data-layout="bubble"][data-expanded="false"] {
````

The scope of this PR is to address the position of only timestamp wrapper inside `.mx_EventTile.mx_EventTile_info[data-layout="bubble"]` (the timestamp at the start of the line displayed on screenshots), because changing the `left` value for all of those selectors at once can cause a regression somewhere and makes it really difficult to troubleshoot as there are not tests to cover all of them yet). Until tests are implemented for the area, I think it should be safe to change the value conservatively.

For me it is not clear where the timestamp styled with `mx_MessageActionBar + .mx_MessageTimestamp` is displayed, therefore for now I would like to leave it as it is, which is `left: -77px` on L529 below.

Since the selector `.mx_EventTile.mx_EventTile_info[data-layout="bubble"]` is same as `.mx_EventTile[data-layout="bubble"].mx_EventTile_info`, it can be safely nested inside `.mx_EventTile[data-layout="bubble"]`.

Reply:
Well, adding tests which cover these areas should be done before changing style rules I guess.

Changing values, how small the change might be, can cause regression which is hard to detect, and I want to avoid that.